app/src/main/java/com/napzak/market/presentation/marketinfo/MarketInfoRoute.kt

```diff
@@ -0,0 +1,335 @@
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.napzak.market.R
+import com.napzak.market.core.common.extension.noRippleClickable
+import com.napzak.market.core.common.state.UiState
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.core.type.SortType
+import com.napzak.market.core.type.TradeType
+import com.napzak.market.domain.genre.model.Genre
+import com.napzak.market.presentation.explore.type.ExploreBottomSheetType
+import com.napzak.market.presentation.marketinfo.component.MarketFilterGroup
+import com.napzak.market.presentation.marketinfo.component.MarketInfoBottomSheetScreen
+import com.napzak.market.presentation.marketinfo.component.MarketInfoTopSection
+import com.napzak.market.presentation.marketinfo.component.MarketProductListSection
+import com.napzak.market.presentation.marketinfo.component.MarketTradeTypeTab
+import com.napzak.market.presentation.marketinfo.state.MarketInfoBottomSheetState
+import com.napzak.market.presentation.marketinfo.state.MarketInfoUiState
+import com.napzak.market.presentation.marketinfo.state.MarketUiInformation
+import kotlin.String
+
+@Composable
+fun MarketInfoRoute(
+    storeId: Long,
+    onProductDetailNavigate: () -> Unit,
+    onBackButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    viewModel: MarketInfoViewModel = hiltViewModel(),
+) {
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+    val bottomSheetState by viewModel.bottomSheetState.collectAsStateWithLifecycle()
+
+    viewModel.setStoreId(storeId)
+
+    LaunchedEffect(uiState) {
+        viewModel.getMarketInformation()
+    }
+
+    MarketInfoScreen(
+        modifier = modifier,
+        uiState = uiState,
+        bottomSheetState = bottomSheetState,
+        onBackButtonClick = onBackButtonClick,
+        onTradeTypeClick = viewModel::updateTradeType,
+        onGenreListClick = {
+            viewModel.initGenreList()
+            viewModel.updateBottomSheetVisibility(ExploreBottomSheetType.GENRE_SEARCHING)
+        },
+        onSoldOutClick = { viewModel.updateSoldOut() },
+        onUnopenClick = { viewModel.updateUnopen() },
+        onSortButtonClick = { viewModel.updateBottomSheetVisibility(ExploreBottomSheetType.SORT) },
+        onItemClick = { onProductDetailNavigate() },
+        onLikeClick = viewModel::updateItemLikeButton,
+        onDismissRequest = { viewModel.updateBottomSheetVisibility(it) },
```

Comment:
p2: 다음과 같이 작성할 수 있습니다!
```suggestion
        onDismissRequest = viewModel::updateBottomSheetVisibility,
```